command/cp.go

```diff
@@ -459,11 +471,22 @@ func (c Copy) prepareDownloadTask(
 	isBatch bool,
 ) func() error {
 	return func() error {
-		dsturl, err := prepareLocalDestination(ctx, srcurl, dsturl, c.flatten, isBatch, c.storageOpts)
+		dsturl, err := prepareLocalDestination(ctx, srcurl, dsturl, c.flatten, c.isSplit, c.storageOpts)
```

Comment:
changing the `isBatch` for `c.isSplit` will compile, but I don't think the `prepareLocalDestination` function is going to do the right thing

Reply:
Yep, good catch